plugins/katello/nightly/installation/index.md

```diff
@@ -141,6 +141,39 @@ These may be set as command line options or in the answer file (/etc/foreman-ins
 foreman-installer --scenario katello <options>
 {% endhighlight %}
 
+## Multiple subnets and domains
+
+Installer only supports one subnet and one DNS domain. Multiple entries can be
```

Comment:
The installer only supports one subnet and one DNS domain via command line arguments.